modules/tinymce/src/themes/silver/test/ts/browser/editor/contextmenu/DesktopContextMenuTest.ts

```diff
@@ -170,23 +160,4 @@ describe('browser.tinymce.themes.silver.editor.contextmenu.DesktopContextMenuTes
     pressDownArrowKey();
     await pAssertFocusOnItem('Delete Table', '.tox-collection__item:contains("Delete table")');
   });
-
-  it('TBA: Test context menus on placeholder image inside a table', async () => {
```

Comment:
Sorry if my previous comment wasn't clear, but this also shouldn't have been removed.